simulation_utils/ParameterParser.h

```diff
@@ -68,8 +68,8 @@ class Simulation_Parameters {
     /* Simulation Settings */ 
     double MAX_TIME;                   // [s]
     double dt;                         // [s]
-    double earliest_sim_start_unix;    // [s] Next three time variables are measured relative to J2000
-    double latest_sim_start_unix;      // [s]
+    double earliest_sim_start_J2000;    // [s] Next three time variables are measured relative to J2000
+    double latest_sim_start_J2000;      // [s]
     double sim_start_time;             // [s] 
```

Comment:
is `sim_start_time` also in J2000? or should the comment read "next two"

Reply:
Yes sim_start_time is in J2000 as well. I did not rename that because its not a parameter and is something to be dispersed